include/NeoFOAM/fields/field.hpp

```diff
@@ -110,21 +155,7 @@ class Field
         {
             exit(1);
         }
```

Comment:
```suggestion
        NF_DEBUG_ASSERT(results.size == size_, "Parsed Field size not the same as current field size");
```
Sorry sneaking in a change here - feel free to discard.